app/webpack.config.js

```diff
@@ -0,0 +1,12 @@
+const merge = require('webpack-merge');
+const baseConfig = require('./webpack/base.config');
+const prodConfig = require('./webpack/prod.config');
+const devConfig = require('./webpack/dev.config');
+
+module.exports = () => {
+  const mode = process.env.NODE_ENV;
```

Comment:
With `--env.NODE_ENV=development` webpack option you will get access to `NODE_ENV` as a property of function parameters object